crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -795,6 +795,18 @@ pub fn tile_config(node: &Node) -> TileConfig {
     TileConfig::new(repeat)
 }
 
+fn extract_attr_value_i64(node: &Node, key: &str) -> i64 {
+    let value = node.attrs.get(key).unwrap().clone().into_i64();
+    value
+}
+
+/// Create a TopKConfig from the attributes of the node.
+pub fn top_k_config(node: &Node) -> TopKConfig {
+    let axis: i64 = extract_attr_value_i64(node, "axis");
+    let k: i64 = extract_attr_value_i64(node, "k");
+    TopKConfig::new(axis, k)
+}
```

Comment:
I added support for the later versions however, it seems I can't test them because with the code as is. The tests require the input to be initialized with a value and doing so makes the IR not have input anymore. For example, I get the following error after initializing:

```
No variable with name input1
```

I'm incrementally pushing commits, so I'll just resolve all the comments when I finish everything.